src/deposit.cairo

```diff
@@ -51,17 +69,18 @@ mod Deposit {
         decimals_difference: DecimalScale,
```

Comment:
Don't quite understand why this needs to be named as `decimals_dfiference`, because when this function is called, `deposit_token_decimals` is the variable that is being used for this.